packages/@sanity/form-builder/src/inputs/Image/ImageInput.js

```diff
@@ -227,22 +245,31 @@ export default class ImageInput extends React.PureComponent<*> {
 
   renderImageTool() {
     const {value} = this.props
-    const hotspot = (value && value.hotspot) || DEFAULT_HOTSPOT
-    const crop = (value && value.crop) || DEFAULT_CROP
 
-    const {uploadingImage, materializedImage} = this.state
+    const hotspot = (this.state.value && this.state.value.hotspot) || (value && value.hotspot) || DEFAULT_HOTSPOT
+    const crop = (this.state.value && this.state.value.crop) || (value && value.crop) || DEFAULT_CROP
+
+    const {uploadingImage} = this.state
 
-    const imageUrl = uploadingImage ? uploadingImage.previewUrl : (materializedImage || {}).url
+    const imageUrl
+      = uploadingImage
```

Comment:
A bit weird formatting here? Probably should be like:

```
const imageUrl = uploadingImage
  ? uploadingImage.previewUrl
  :  getImageBuilder()
       .image(value)
       .width(800)
       .ignoreImageParams()
       .url()
```

Reply:
👍